cmd/kubeadm/app/constants/constants.go

```diff
@@ -177,6 +177,10 @@ const (
 
 	// APICallRetryInterval defines how long kubeadm should wait before retrying a failed API operation
 	APICallRetryInterval = 500 * time.Millisecond
+	// APICallWithWriteTimeout specifies how long kubeadm should wait for api calls with at least one write
+	APICallWithWriteTimeout = 40 * time.Second
+	// APICallWithReadTimeout specifies how long kubeadm should wait for api calls with only reads
+	APICallWithReadTimeout = 15 * time.Second
```

Comment:
ref: https://github.com/kubernetes/kubeadm/issues/2092#issuecomment-608394171

Reply:
maybe we shouldn't add APICallWithReadTimeout if it's not used yet?
(just a question at this point)

Reply:
@neolit123 I am ok with either putting the const here or not... I put it here just for other contributors, so that they will not have to search the GitHub for the value of this const.

Reply:
yeah, just a generic question for everyone at this point.